code/game/machinery/Sleeper.dm

```diff
@@ -354,7 +354,7 @@
 				usr << "[G:affecting.name] will not fit into the sleeper because they have a slime latched onto their head."

```

Comment:
probably goes past the scope of this PR but this is a painful abuse of colons here